src/net/sourceforge/zmanim/hebrewcalendar/Daf.java

```diff
@@ -25,13 +25,13 @@
 	private int masechtaNumber;
 	private int daf;
 
-	private static String[] masechtosBavliTransliterated = { "Berachos", "Shabbos", "Eruvin", "Pesachim", "Shekalim",
+	private static final String[] masechtosBavliTransliterated = { "Berachos", "Shabbos", "Eruvin", "Pesachim", "Shekalim",

```

Comment:
@BehindTheMath, I have a ToDo (sadly not in the code) to allow users to set different names for transliterated (such as Sephardi transliteration. While I do not yet have the set() methods, I will add them. Do you see any need for anyone to ever change the Hebrew ones? I can't see a reason.

Reply:
>Do you see any need for anyone to ever change the Hebrew ones? I can't see a reason.

I don't either.

Reply:
Do you want me to remove this change? As the code stands now, there's no point in having them `private` and not `final` without setters.

Reply:
@BehindTheMath ,
Can you please rebase this against the latest master? I do ask that you remove the final on the masechtosBavliTransliterated and masechtosYerushalmiTransliterated ? Since I plan to add a setter/getter, I want to avoid adding a final just to revoke it a drop later.
Thanks

Reply:
Done.